src/resources/js/tickets.js

```diff
@@ -393,22 +395,32 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 						$( document.getElementById( 'rsvp_ticket_stock_total_value' ) ).text( responseData.ticket_stock );
 
 						// remove old ticket table
-						var ticketTable = document.getElementById('ticket_list_wrapper');
-						ticketTable.innerHTML = '';
+						var $ticket_table = $( document.getElementById( 'ticket_list_wrapper' ) );
+
+						if ( 0 === $ticket_table.length ) {
+							// if it's not there, create it :(
+							var $container = $( '.tribe_sectionheader.ticket_list_container' );
+							$ticket_table = $( '<div>', {id: "ticket_list_wrapper"});
+							container.append( ticket_table );
```

Comment:
above this line i only see `$ticket_table` — with the `$`, but here i see `ticket_table` ... does `ticket_table` (no `$`) exist? if so, ignore this question/comment